webrender/src/debug_render.rs

```diff
@@ -389,8 +389,11 @@ impl DebugRenderer {
 
             // Triangles
             if !self.tri_vertices.is_empty() {
+                #[cfg(feature = "gl")]
                 device.bind_program(&self.color_program);
                 device.set_uniforms(&self.color_program, &projection);
+                #[cfg(not(feature = "gl"))]
+                device.bind_program(&self.color_program);
```

Comment:
looks like we are doing the same thing for GL and non-GL now, can we remove the duplication?